db/migrations/20210222113222_create_orders_table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function (knex) {
+  console.log('creating orders table');
+  return knex.schema.createTable('orders', (ordersTable) => {
+    ordersTable.increments('order_id').primary();
+    ordersTable.integer('table_id').references('tables.table_id').notNullable();
+    ordersTable
+      .integer('food_item_id')
+      .references('food_items.food_item_id')
+      .notNullable();
+    ordersTable.string('description');
```

Comment:
What is an order description?

Reply:
order description is in place of the name column for orders which was in the schema, i.e. "mains orders for table 13"